Data/ByteString/Internal.hs

```diff
@@ -393,6 +396,8 @@ createUptoN l f = do
 {-# INLINE createUptoN #-}
 
 -- | Create ByteString of up to size @l@ and use action @f@ to fill it's contents which returns its true size.
```

Comment:
This comment does not quite make sense to me grammatically. At the very least "it's -> its" and probably "of up to -> up to", but it is also unclear who returns its (btw whose?) size: `createUpToN'`, action `f` or its `contents.

Reply:
Could you take another look?